test/unit/test-amp-story-player.js

```diff
@@ -239,4 +249,34 @@ describes.realWin('AmpStoryPlayer', {amp: false}, (env) => {
       expect(iframes[1].getAttribute('i-amphtml-iframe-position')).to.eql('0');
     });
   });
+
+  describe('Cache URLs', () => {
```

Comment:
Can you add a case with an invalid cache URL?

Reply:
I spent a lot of time trying to write it, but couldn't get it to work :( ... See line 282, I'm getting `expected [Function] to throw an error` when an error is actually being thrown when running the tests... do you have any suggestions? we can discuss offline.

Reply:
Done.